ngraph/core/reference/include/ngraph/runtime/reference/utils/span.hpp

```diff
@@ -92,6 +92,34 @@ namespace ngraph
                     return {m_data + offset, std::min(size, m_size - offset)};
                 }
 
+                Span& drop_front(std::size_t number_of_elements)
```

Comment:
What do you think of setting the default `number_of_elements` value to 1?

Reply:
I prefer to be explicit during the call so I won't default here.